### ISSUE TITLE
📜Create guide for rendering text

### ISSUE THREAD

**Issue by CalvinWilkinson** [KinsonDigital/VelaptorDocs#117]
### Complete The Item Below

- [X] I have updated the title without removing the 📜 emoji.

### Description

Create a guide on how to render text.

### Here are the topics that should be covered:

1. Rendering text
2. Rendering multi-line text
3. Make sure that all of the other versions have the same guide.

### Acceptance Criteria

- [x] Guide created
- [x] Content Covered
- [x] All of the other versions have the same guide
- [x] The _**HelloVelaptor**_ project renamed to _**RenderText**_
- [x] The tag recreated to point to the new _**HelloVelaptor**_ project changes

### ToDo Items

- [X] Change type labels added to this issue.  Refer to the _**Change Type Labels**_ section below.
- [x] Priority label added to this issue.  Refer to the _**Priority Type Labels**_ section below.

### Issue Dependencies

- KinsonDigital/Velaptor#752
    - This improvement needs to be done before this guide can be created.  We plan on releasing the new _**Velaptor**_ font loader changes at the same time we release the rendering font/text guide.

### Additional Information:

**_<details closed><summary>Priority Type Labels</summary>_**

| Priority Type       | Label                                                                  |
|---------------------|------------------------------------------------------------------------|
| Low Priority        | https://github.com/KinsonDigital/VelaptorDocs/labels/low%20priority    |
| Medium Priority     | https://github.com/KinsonDigital/VelaptorDocs/labels/medium%20priority |
| High Priority       | https://github.com/KinsonDigital/VelaptorDocs/labels/high%20priority   |
</details>

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct.
